tools/generate_hotkeys.py

```diff
@@ -26,14 +35,33 @@
             "|Command|Key Combination|\n"
             "| :--- | :---: |\n"
         )
+        keys_list = []
```

Comment:
If this is just for checking duplicates, please name it more specifically than this.

Reply:
Not sure I like the new name. Can't think of a better one right now though.